cmd/frontend/graphqlbackend/org.go

```diff
@@ -283,6 +283,33 @@ func (r *schemaResolver) UpdateOrganization(ctx context.Context, args *struct {
 	return &OrgResolver{db: r.db, org: updatedOrg}, nil
 }
 
+// RemoveOrganization removes an org and all resources associated to it.
+func (r *schemaResolver) RemoveOrganization(ctx context.Context, args *struct {
+	Organization graphql.ID
+}) (*EmptyResponse, error) {
+	if !envvar.SourcegraphDotComMode() {
+		return nil, errors.New("hard deleting organization is not supported")
```

Comment:
nit: this would the error message a bit more clear, since currently it is identical to the error message of the feature flag not enabled.
```suggestion
		return nil, errors.New("hard deleting organization is only supported on Sourcegraph.com")
```